common/util.go

```diff
@@ -26,6 +28,9 @@ import (
 // HTTPTimeout timeout used in HTTP connections between nodes
 var HTTPTimeout = 8 * time.Second
 
+// Max Segment Size (cap reading HTTP response body at this size)
+const MaxSegSize = 20 * 1024 * 1024 // 20 MiB - 20s of 1080p30fps video @ 8000kbps
```

Comment:
The max duration of a segment is currently [5 minutes](https://github.com/livepeer/go-livepeer/blob/ffabc358fe3115c81e0fb6fc83f1b6b281194498/server/broadcast.go#L32). Support for segments up to this max duration was added awhile back. I'm not sure how common longer segments as a use case are today (they came up while experimenting with preliminary VOD support + mp4 transcoding), but I think we should set this byte size limit based on the current duration limit.